release/src/version-helpers.ts

```diff
@@ -99,3 +99,41 @@ export const isLatestVersion = (thisVersion: string, allVersions: string[]) => {
 
   return compareVersions(String(coerce(thisVersion.replace(/(v1|v0)\./, ''))), lastVersion) > -1;
 };
+
+export const getNextVersions = (versionString: string): string[] => {
+  if (!isValidVersionString(versionString)) {
+    throw new Error(`Invalid version string: ${versionString}`);
+  }
+
+  const versionType = getVersionType(versionString);
+
+  if (versionType === 'rc' || versionType === 'patch') {
+    return [];
+  }
+
+  const editionString = isEnterpriseVersion(versionString)
+    ? 'v1.'
+    : 'v0.';
+
+  // minor releases -> next minor release
+  const [major, minor] = versionString
+    .replace(/(v1|v0)\./, '')
+    .replace(/.0$/, "")
+    .split('.')
+    .map(Number);
+
+  if (versionType === 'minor') {
+    return [editionString + [major, minor + 1].join('.')];
+  }
+
+
+  // major releases -> x.1 minor release AND next .0 major release
+  if (versionType === 'major') {
+    return [
+      editionString + [major, 1].join('.'),
+      editionString + [major + 1, 0].join('.'),
```

Comment:
This is not matching the past major milestones that didn't have `.0` in the end.
However, it matches the release formats that DO have it, so I think it's even more correct the way you implemented it now.

Nothing is stopping us from updating the major milestone format going forward and, frankly, nobody will even notice the difference.